environment.py

```diff
@@ -0,0 +1,127 @@
+import os
+
+import time
+
+from color_refinement import get_number_automorphisms, is_isomorphisms
+from graph_io import load_graph
+
+PATH_BRANCHING = 'graphs/branching'
+PATH_COLORREF = 'graphs/colorref'
+PATH_TREEPATHS = 'graphs/treepaths'
+OUTPUT_PATH = 'export/result'
+TIME_ROUND = 5
+
+FILE = 'graphs/colorref/colorref_smallexample_4_16.grl'
+
+
+def main():
+    if FILE == '':
+        graph_files = find_graphs()
+    else:
+        graph_files = [FILE]
+    process_graph_files(graph_files)
+
+
+def find_graphs():
+    branching_graphs = [PATH_BRANCHING + '/' + file for file in os.listdir(PATH_BRANCHING)]
+    colorref_graphs = [PATH_COLORREF + '/' + file for file in os.listdir(PATH_COLORREF)]
+    treepath_graphs = [PATH_TREEPATHS + '/' + file for file in os.listdir(PATH_TREEPATHS)]
+    return branching_graphs + colorref_graphs + treepath_graphs
+
+
+def process_graph_files(graph_files):
+    for file in graph_files:
+        graphs = get_graphs_from_file(file)
+        process_graphs(file, graphs)
+
+
+def get_graphs_from_file(file):
+    with open(file) as f:
+        L = load_graph(f, read_list=True)
+    return L[0]
+
+
+def process_graphs(file, graphs):
+    iso_start = time.time()
+    isomorphs = calculate_isomorphs(graphs)
```

Comment:
Het is vermoedelijk sneller om eerst de automorphisms uit te rekenen.

Reply:
Nou.. als je eerst de automorfismen uitrekent, ga je dat voor allemaal doen.. Maar als je eerst de isomorfismen doet hoef je alleen maar de automorfismen van 1 van de isomorfismen uit te rekenen.. Of snap ik nu iets niet?